sources/packages/backend/apps/db-migrations/src/sql/Reports/Update-data-inventory-report.sql

```diff
@@ -0,0 +1,527 @@
+            left join  (
+                select
+                	ds.student_id,
+                	ds.disbursement_value_code,
+                    SUM (ds.overaward_value) as overaward_value
+                from
+                    sims.disbursement_overawards ds
+                where ds.disbursement_value_code = ''CSLF''
+                group by
+                    ds.student_id,
+                    ds.disbursement_value_code
+            ) as cslf_disbursement_overawards on cslf_disbursement_overawards.student_id = students.id
+            left join  (
+                select
+                	ds.student_id,
+                	ds.disbursement_value_code,
+                    SUM (ds.overaward_value) as overaward_value
+                from
+                    sims.disbursement_overawards ds
+                where ds.disbursement_value_code = ''CSLP''
+                group by
+                    ds.student_id,
+                    ds.disbursement_value_code
+            ) as cslp_disbursement_overawards on cslp_disbursement_overawards.student_id = students.id
+            left join  (
+                select
+                	ds.student_id,
+                	ds.disbursement_value_code,
+                    SUM (ds.overaward_value) as overaward_value
+                from
+                    sims.disbursement_overawards ds
+                where ds.disbursement_value_code = ''BCSL''
+                group by
+                    ds.student_id,
+                    ds.disbursement_value_code
+            ) as bcsl_disbursement_overawards on bcsl_disbursement_overawards.student_id = students.id
+            left join (
+            	select a.id as application_id, count(1)
+				from
+					sims.applications a,
+					jsonb_array_elements(a.data->''dependants'') b
+				where (b->>''validDependent'')::integer <= 11 or jsonb_array_length(b->''pdDependentUpload'') > 0
+				group by a.id
+            ) as dependant_11_yo_or_disabled on dependant_11_yo_or_disabled.application_id = applications.id
+            left join (
+            	select a.id as application_id, count(1)
+				from
+					sims.applications a,
+					jsonb_array_elements(a.data->''dependants'') b
+				where (b->>''validDependent'')::integer > 11 or a.data ->> ''supportnocustodyDependants'' = ''yes''
+				group by a.id
+            ) as dependant_12_yo_or_other on dependant_12_yo_or_other.application_id = applications.id
+            left join sims.disbursement_schedules disbursement_schedules on disbursement_schedules.student_assessment_id = current_assessment.id
+           	left join (
+            	select su.id, count(1)
+				from
+					sims.supporting_users su,
+					jsonb_array_elements(su.supporting_data->''parentDependentTable'') b
+				where (b->>''attendingPostSecondarySchool'')::text = ''yes''
+				group by su.id
+            ) as parent1_dependant_number_post_sec_school on parent1_dependant_number_post_sec_school.id = parent1.id
```

Comment:
Some variables like `parent1_dependant_number_post_sec_school` will be likely calculated by Camunda. If not, it would be good to have Camunda inspect the data and execute any dynamic data-related calculations.
It can have it captured in the ticket and planned for an upcoming effort.

Reply:
To be clear, I would consider the decision and implementation outside the scope of this PR, IMO.